worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/WorldGuardPlugin.java

```diff
@@ -111,6 +113,9 @@ public class WorldGuardPlugin extends JavaPlugin {
 
     private static final int BSTATS_PLUGIN_ID = 3283;
 
+
+    public FoliaLib foliaLib;
```

Comment:
do not use public fields

Reply:
This has been resolved in the latest pull from @Loving11ish